packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -132,11 +135,11 @@ export default class VscodeHatRenderer {
     if (hatsDir) {
       await this.updateShapeOverrides(hatsDir);
 
-      if (fs.existsSync(hatsDir)) {
+      fs.access(hatsDir).then(() =>
         this.hatsDirWatcherDisposable = watchDir(hatsDir, () =>
           this.updateShapeOverrides(hatsDir),
-        );
-      }
+        ))
+        .catch(() => console.error("cannot watch hatsDir " + hatsDir));
```

Comment:
Note to reviewers: I'm not entirely sure this change is correct.

Reply:
also, are you sure it's ok to use `fs` directly? I'd think we would need to use the `vscode` fs stuff exclusively, like we do everywhere else in this file

Reply:
I thought I tried `await fs.access` and it didn't work/compile for some reason. I'll try again.

Yes, ideally we will remove all direct calls to `fs`. This PR does not completely port VscodeHatRenderer to the web runtime, it does just enough to make the hat images load. The remaining calls to fs are currently implemented by a shim library that just fails; so in the access/watchDir case, no watching happens, which is fine for now.

Also, of course, there is no watchDir equivalent vscode.workspace.fs (because HTTP). Ultimately this code, along with a variety of other cases, should be factored out into a native-runtime-specific injected dependency so that it can work locally and not throw errors in on web.

Reply:
The issue with `fs.access` is that it returns a `Promise<void>`, not a `Promise<boolean>`. The alternative to then/catch is try/catch.  See https://stackoverflow.com/questions/73674525/node-js-fs-access-returns-undefined-no-matter-what.

Reply:
Yes I'd use try/catch